test/over_react_test/custom_matchers_test.dart

```diff
@@ -485,7 +490,13 @@ main() {
           });
 
           test('when passed in a matcher looking for multiple logs', () {
-            shouldPass(logs, logsToConsole(containsAll([contains('random log1'), contains('combination error')])));
+            final expectedLogMatchers = [contains('random log1')];
+
+            if (runtimeSupportsPropTypeWarnings()) {
+              shouldPass(logs, logsToConsole(containsAll([...expectedLogMatchers, contains('combination error')])));
+            } else {
+              shouldPass(logs, logsToConsole(containsAll(expectedLogMatchers)));
+            }
```

Comment:
Similar to the comment above about the prop error in the list literal being arbitrary, since this test was just verifying the matcher could find two logs in a list of arbitrary logs I'd expect it to be valid. Let me know if I'm missing something though!

Reply:
Addressed in https://github.com/Workiva/over_react_test/pull/104/commits/5e736cd83aa9091f0e0dd699c9e3d8020ce7fd4f